packages/client3/pages/artist/dashboard/index.page.tsx

```diff
@@ -50,20 +59,73 @@ function DashBoard() {
         <DashboardTotal
           totalEarnedAmount={dashBoardData.totalEarnedAmount}
           totalRaisedAmount={dashBoardData.totalRaisedAmount}
+          onCreatePoolClicked={() => {
+            /** */
+          }}
         />
-        <Heading className="mt-10 mb-5 text-white" level={5}>
-          My Open Pool
-        </Heading>
-        <ArtistPool
+        <div className="mb-5 mt-10 flex">
+          <Heading className="flex-1 text-white" level={5}>
+            My Open Pools
+          </Heading>
+          <Caption className="mr-11 w-60 flex-none text-right">
+            Progress
+          </Caption>
+        </div>
+        {/* <ArtistPool
           isVerified={isVerified}
           onButtonClick={() => {
             //TODO: Dummy function definition below
             if (isVerified) {
-              setDashBoardData(DummyDashboardData);
+              setDashboardData(DummyDashboardData);
             } else {
               setIsVerified(true);
```

Comment:
I know this isn't a part of your PR, but can you pull a `// Todo KYC integration` here? I think this shouldn't be setting their verification to `true` rather direct them to KYC